lib/puma/error_logger.rb

```diff
@@ -0,0 +1,92 @@
+# frozen_string_literal: true
+
+require 'puma/const'
+
+module Puma
+  # The implementation of a logging in debug mode.
```

Comment:
Comment here is out of date now - I think it is uneccessary anyway and can just be deleted.

Reply:
> Basically nothing about the request should ever be logged unless the user opted in via PUMA_DEBUG.

Agree, so I made the interface to have 2 methods: `debug` and `info`. Obviously, `info` prints only non-sensitive info and `debug` prints only in debug mode.